ush/detect_machine.sh

```diff
@@ -33,6 +33,10 @@ case $(hostname -f) in
   hfe1[0-2]) MACHINE_ID=hera ;; ### hera10-12
   hecflow01) MACHINE_ID=hera ;; ### heraecflow01
 
+  hfe0[1-9]) MACHINE_ID=ursa ;; ### ursa01-09
+  hfe1[0-6]) MACHINE_ID=ursa ;; ### ursa10-16
```

Comment:
Are these correct?

Reply:
No, they are not.  Thanks for the catch.

Reply:
Fixed.